module/zfs/qat_compress.c

```diff
@@ -275,20 +284,20 @@ qat_fini(void)
 	}
 }
 
-int
-use_qat(size_t s_len)
+boolean_t
+qat_should_use(size_t s_len)
 {
-	if (zfs_qat_disable == B_TRUE ||
-	    qat_init_done == 0 ||
+	if (zfs_qat_disable ||
+	    !qat_init_done ||
 	    s_len < QAT_MIN_BUF_SIZE ||
 	    s_len > QAT_MAX_BUF_SIZE) {
-		return (0);
+		return (B_FALSE);
 	}
-	return (1);
+	return (B_TRUE);
```

Comment:
this can be revised as:
```
boolean_t qat_should_use(size_t s_len)
{
        return (!zfs_qat_disable && qat_init_done &&
            s_len >= QAT_MIN_BUF_SIZE && s_len <= QAT_MAX_BUF_SIZE);
}
```